x/sentry_integration.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2017-2018 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package x
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/golang/glog"
+	"github.com/mitchellh/panicwrap"
+)
+
+var env string
+
+// InitSentry initializes the sentry machinery.
+func InitSentry(ee bool) {
+	if ee {
+		env = "enterprise"
+	} else {
+		env = "oss"
+	}
+	initSentry()
+}
+
+func initSentry() {
+	if err := sentry.Init(sentry.ClientOptions{
+		Dsn:              "https://58a035f0d85a4c1c80aee0a3e72f3899@sentry.io/1805390",
```

Comment:
I'm confused. Why isn't there a way to change Sentry's DSN so I can get notified when a panic occurs?
Or is this feature meant only for developers (Dgraph Labs)?